File: Web/src/app/models/usuario.ts
export class Usuario {
  id: any;
  nome: string;
  login: string;
  password: string;
  perfil: UsuarioPerfil;
  ativo: boolean;
  token?: string;
}

export enum UsuarioPerfil {
  Garcon = 1,
  Cozinheiro = 2,
  Administrador = 3,
}

File: favodemel-api/src/FavoDeMel.Domain/Entities/Usuarios/Usuario.cs
using FavoDeMel.Domain.Common;
using System;

namespace FavoDeMel.Domain.Entities.Usuarios
{
    public class Usuario : Entity<Guid>
    {
        public string Nome { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        public UsuarioPerfil Perfil { get; set; }
        public decimal Comissao { get; set; }
        public bool Ativo { get; set; }
    }
}
